input/ignoreWarnings.txt

```diff
@@ -219,3 +219,14 @@ The definition for the Code System with URI 'urn:oid:2.16.840.1.113883.6.238' do
 
 # SECTION 2 - INFORMATION - The following messages are generated from code systems and value sets currently in 'draft' status within the inherited/base resource snapshot.
 Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/discharge-disposition|1.0.0
+
+# The following are confirmed for suppression based on US Core:
```

Comment:
The comment "The following are confirmed for suppression based on US Core:" is too general and is not helpful. Please provide more detailed description as reasons why these detailed race codes error were suppressed and point to the Zulip thread where this specific issue are discussed: "https://chat.fhir.org/#narrow/stream/179252-IG-creation/topic/Publisher.20reporting.20code.20not.20found/near/460123037"